server.js

```diff
@@ -0,0 +1,62 @@
+// Load Environment Variables from the .env file
+require('dotenv').config();
+
+// Application Dependencies
+const express = require('express');
+const cors = require('cors');
+
+// Application Setup
+// - make an express app!
+const app = express();
+// - get the port on which to run the server
+const PORT = process.env.PORT || 3000;
+// - enable CORS
+app.use(cors());
+const geoData = require('./data/geo.json');
+const weatherData = require('./data/weather.json');
+
+app.get('/location', (req, res) => {
+    try {
+        const object = formatObject();
+        res.json(object);
+    }
+    catch(err) {
+        res.status(500).send('Sorry something went wrong');
+    }
+});
+app.get('/weather', (req, res) => {
+    const object = formatWeatherObject();
+    console.log(object);
+    res.json(object);
+    
+});
+
+app.listen(PORT, () => console.log('listening on 3001'));
+
+function formatObject(res) {
+    if(res !== 'Portland') {
```

Comment:
This is why your location route isn't working. Remove this, and the res parameter from the function and your route works.